src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -254,5 +427,63 @@ public static bool IsHexLowerChar(int c)
             0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, // 239
             0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF  // 255
         };
+
+        private static ReadOnlySpan<byte> LowerHexLookupTable => new byte[]
+        {
+            0x30, 0x31, 0x32, 0x33, 0x34, 0x35, 0x36, 0x37, 0x38, 0x39, 0x61, 0x62, 0x63, 0x64, 0x65, 0x66,
```

Comment:
** not relevant yet -- only when SSE2 comes into play **

Had a quick look at HexMate, and saw that (as expected) the constants for AVX2 just doubles SSE2 in the upper lane.

So in order to save some (disk) space for the assembly, one could use this outline:
```c#
Vector128<byte> sseConstant = ReadVector(Sse2ContantData);
Vector256<byte> avxConstant = Vector256.Create(sseConstant, sseConstant);
```

In essence it's about avoiding to have duplicated data in the "text-section" of the DLL.

Reply:
Good point, I added your suggestion to the HexMate repo ([here](https://github.com/tkp1n/HexMate/commit/14cfb756e534d7c2e4de9b69d252a4d711a132f7)) 👍

I let this open as a reminder in case we get to SSE2 in this PR.